.jenkinsci/artifacts.groovy

```diff
@@ -11,17 +11,26 @@ def uploadArtifacts(filePaths, uploadPath, artifactServers=['artifact.soramitsu.
   }
   def shaSumBinary = 'sha256sum'
   def md5SumBinary = 'md5sum'
+  def gpgKeyBinary = 'gpg --armor --detach-sign --default-key 9C49B055 --batch --yes --passphrase-fd 0'
   if (agentType == 'Darwin') {
     shaSumBinary = 'shasum -a 256'
     md5SumBinary = 'md5 -r'
   }
   sh "> \$(pwd)/batch.txt"
-  filePathsConverted.each {
-    sh "echo put ${it} $uploadPath >> \$(pwd)/batch.txt;"
-    sh "$shaSumBinary ${it} | cut -d' ' -f1 > \$(pwd)/\$(basename ${it}).sha256"
-    sh "$md5SumBinary ${it} | cut -d' ' -f1 > \$(pwd)/\$(basename ${it}).md5"
-    sh "echo put \$(pwd)/\$(basename ${it}).sha256 $uploadPath >> \$(pwd)/batch.txt;"
-    sh "echo put \$(pwd)/\$(basename ${it}).md5 $uploadPath >> \$(pwd)/batch.txt;"
+
+  withCredentials([file(credentialsId: 'ci_gpg_privkey', variable: 'CI_GPG_PRIVKEY'), string(credentialsId: 'ci_gpg_masterkey', variable: 'CI_GPG_MASTERKEY')]) {
+
+    sh "gpg --allow-secret-key-import --import ${CI_GPG_PRIVKEY}"
```

Comment:
`--allow-secret-key-import` is an obsolete as stated in `man`. Should we really use it?